### PR TITLE
[DOCS] update README.md

### LINKED ISSUE #52: [DOCS] improve README.md file
Use your creativity and make improvements to the readme.md file

### PR DESCRIPTION
**Pull request by RakeshSangem**
### Fixes Issue
added new buttons for tech stack and also wrote text for connect with us.

### Screenshots
Screenshot 2022-09-04 182446

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,18 +6,21 @@ A web app where you can find contributors for your open-source project or as an
...
+![Firebase](https://img.shields.io/badge/Firebase-305FCB?style=for-the-badge&logo=firebase&logoColor=orange)
+![MongoDB](https://img.shields.io/badge/MongoDB-305FCB?style=for-the-badge&logo=mongodb&logoColor=white)
 
-1. Frontend - Next.js
-2. CSS Framework - Tailwind
-3. Backend - NextJS APIs + MongoDB
+
+1. Frontend - **Next.js**
+2. CSS Framework - **Tailwind CSS**
+3. Backend - **NextJS APIs + MongoDB**
```
Comment (rohitdasu) [inline README.md:23]: add firebase here in backend section

File: README.md [hunk 2]
```diff
@@ -6,18 +6,21 @@ A web app where you can find contributors for your open-source project or as an
 Find people who have the same vision as you do and contribute to each other for building amazing products.
 
 ## Connect with us
+As a community, we always encourage people to share their thoughts and ideas. Do you want to talk to us? Join our Discord server by clicking the badge below.
 
-[![Discord](https://img.shields.io/badge/-discord-black?style=social&logo=discord)](https://discord.gg/FQtyMWFZQ9)
+[![DISCORD - Join](https://img.shields.io/badge/DISCORD-Join-2ea44f?style=for-the-badge&logo=discord&logoColor=white)](https://discord.gg/9NGVjh9h)
```
Comment (rohitdasu) [inline README.md:11]: is this discord link expires? previous one doesn't expires at all. need a clarification

**Review by rohitdasu — CHANGES_REQUESTED** [review]
please work on the comments

File: README.md (hunk 1 above)
Comment (RakeshSangem) [reply README.md:23]: We are using firebase for Authentication only right?


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Brittle

Comment (rohitdasu) [reply README.md:23]: yes, so?

Comment (rohitdasu) [reply README.md:23]: Firebase is a backend service

File: README.md (hunk 2 above)
Comment (RakeshSangem) [reply README.md:11]: I don't have idea regarding that one. I have created this link though invite option in the discord.

Comment (rohitdasu) [reply README.md:11]: then please keep the older link.

**Comment by RakeshSangem** [comment]
Have made changes. Is it fine now?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Review by rohitdasu — APPROVED** [review]
It's better now, but needs more improvement that can be done later 👍

**Pull request merged**